src/shared/services/fields/fields.utils.ts

```diff
@@ -127,6 +127,9 @@ export class FieldsUtils {
       return page.parsedShowCondition.match(currentState);
     };
   }
+  public getCurrentEventState(eventTrigger, form): any {
```

Comment:
can you change the methods in this class now to call getCurrentEventState? So we remove duplication